oak_functions/loader/src/tests.rs

```diff
@@ -94,7 +96,11 @@ fn bench_wasm_handler(bencher: &mut Bencher) {
         // to be less than a fixed threshold.
         // Note: `summary.mean` is in nanoseconds, even though it is not explicitly documented in
         // https://doc.rust-lang.org/test/stats/struct.Summary.html.
-        assert!(summary.mean < 50_000.0, "elapsed time: {}ns", summary.mean);
+        assert!(
+            summary.mean < 1_000_000.0,
```

Comment:
Now it takes more than 600ns.

Reply:
:grimacing: why is that? There's not much happening, is it?

Reply:
This one is actually reading the request and writing the response. I suppose that is taking some time. The whole thing is still less than 1ms. Is that bad?

Reply:
Ah you are right, I thought it was taking 1s now! Never mind, all good! :)

Reply:
Yeah! I made that mistake before too :D